### PR TITLE
Removes drone fabricator and logs more grabs.

### PR DESCRIPTION
**Pull request by Super3222**
Fixes https://github.com/yogstation13/yogstation/issues/433
Fixes https://github.com/yogstation13/yogstation/issues/466

:cl: Super3222
tweak: Normal grab and Strangling grabs are now logged.
rscdel: Drone Fabricator has been taken off the map.
/:cl:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
These should be separate PRs

**Comment by Super3222** [comment]
Ignore the extra grab logging. If this PR doesn't get merged, I'll move it to another.

**Comment by Nirnael** [comment]
Drones are fine, the ones that grief get punished, I don't think this is the way to go. It hasn't become much of a problem that drones steal everything either. If this gets merged, add drones to mining at least.

**Comment by GrayRachnid** [comment]
What nirn said, I haven't seen or head much about drones that'll make them more trouble than other players. They get complaints now and then, but so do normal players.

**Comment by Super3222** [comment]
Drones don't belong in lavaland.

Anyway, going to leave this up for discussion. I know /tg/ has had their pro/anti drone wars, and Yogs is as equally divided. Let's hear some opinions.

**Comment by ShadowDeath6** [comment]
we can already get drones via the derelict, we don't need hordes of dead players coming back as drones to clean gang tags every time they get shot, it's annoying. robotics can make drones via the exosuit fabricators if that wasn't removed before the drone fabricator, and it wasn't even hard to unlock

**Comment by ParagonPrime** [comment]
It's really hard for drones to return from the derilict. Simply because they have no throw function, thus if they smack into the station at the wrong speed they drift again.

And of course random z levels are shit.

**Comment by ShadowDeath6** [comment]
secret drone teleporter technique

also metal rods can be used to make lattices which is better control than a fire extinguisher

**Comment by TheSadFox** [comment]
> we can already get drones via the derelict, we don't need hordes of dead players coming back as drones to clean gang tags every time they get shot, it's annoying. robotics can make drones via the exosuit fabricators if that wasn't removed before the drone fabricator, and it wasn't even hard to unlock

**Comment by ParagonPrime** [comment]
> > > implying anyone ever built drones back then

**Comment by Kmc2000** [comment]
> It's really hard for drones to return from the derilict. Simply because they have no throw function, thus if they smack into the station at the wrong speed they drift again.

You skrub, you just take rods to build lattices in space

Opinion devalidated as paragoon is obviously a noob drone

**Comment by FluffySurvivor** [comment]
drones are simply grief roles

**Comment by MayhemSailor** [comment]
We ran a vote on the forum when this was being discussed. 25 participated in the vote, 5 abstained, 4 for the removal of the drone dispenser, 16 for the drone dispenser to stay... #willofthepeople

**Comment by Nirnael** [comment]
There were drones in mining before, the idea is to have some drones close, not only the derelict, put them on the station in maintenance then. Also, nobody builds drones, don't kid yourself. As for drones cleaning gang tags, it's up to the council to decide if that's punishable or not since it technically fits their laws, but all it takes is a simple discussion, that's what we have the CM for. Drone fabricator also has a lengthy cooldown, it's only during long rounds that it spawns like 10 drones.

**Comment by sagattyy** [comment]
remove drone fabricator, readd mining drone.

**Comment by ghost** [comment]
Please remove it. I get so many griefing drones who don't read their laws.

**Comment by Nirnael** [comment]
Jobban every griefing drone.

**Comment by TheSadFox** [comment]
> remove drone fabricator, readd mining drone.

What laws and equipment did mining drones even have?

**Comment by ParagonPrime** [comment]
Basically the same as normal drones, exception being that minedrones are allowed to interact with all the mobs in mining and are supposed to mine and return resources and get stuff from RnD.

They were also absolutely ridiculously amazing and a competent minedrone team would bring in an absolute flood of resources for RnD. I once managed to get 40k mining points in one go as one (with advanced plasma cutter, of course)

TL;DR- miners that will care more about actually mining that exploring lavaland

**Comment by Super3222** [comment]
Discouraging miners to mine **EVEN MORE** is definitely the wrong way to go about Lavaland.
And leaving up the 'mining' part of the job to a ghost role is not something I look forward too.

Miners also die a lot on lavaland, and it's basically game over for them unless they wish to switch sides with another "faction" of the game and play from their perspective. Putting mining drones will basically be giving miners extra lives.

Feedback from players that played as a mining drone when we had them? "It was like I was on the asteroid on easy mode."

I really just can't see them in the game. It's just way too easy to play as them. Lavaland is for the miners, and not for the drones. I don't know, that's just my take on mining drones. They had no place after the rebase.

I'm pretty sure that's why no one bothered to re-add them after Groxic confirmed he wasn't going to port them (after the rebase as already over).

**Comment by ShadowDeath6** [comment]
the problem is that no miners ever want to mine because le explore meme

miner drones are probably a good idea since unlike the shitty tg ones they can actually access mining stuff

**Comment by Nirnael** [comment]
The only people that ever bring back minerals are the robust miners that want double mod hyper ka to kill megafauna and get cool loot so miner drones don't sound bad. There probably should be something to discourage them from attacking megafauna but they get shrekt anyway. There would be a few that can't reproduce.

**Comment by Super3222** [comment]
No minedrones, pls.
If you want miners to mine, give them something worth their while.
Otherwise, don't bother making a problem we're already facing even worse.
